hs_core/views/resource_rest_api.py

```diff
@@ -845,15 +851,20 @@ def get(self, request, pk, filename):
         redirect_url = f.url.replace('django_irods/download/', 'django_irods/rest_download/')
         return HttpResponseRedirect(redirect_url)
 
-    def post(self, request, pk):
+    def post(self, request, pk, filename):
```

Comment:
The parameter in the call is templated in urls.py as 'filename' and serves -- in the GET call -- to fetch a file. So, if I rename it, I have to refactor urls.py as well. I would rather have one url pattern that two.